Controller/ApiController.php

```diff
@@ -8,20 +8,30 @@
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\Security\Csrf\CsrfTokenManager;
 
 class ApiController
 {
     /**
      * @var ApiService
      */
     private $service;
-
     /**
-     * @param ApiService $service
+     * @var CsrfTokenManager
```

Comment:
/** @var should be a one line comment

Reply:
Agree but the role of a controller is to validate inputs. Unless there is api controller somewhere is it's good enough for now. I'll add a TODO

Reply:
Refactored as recommended

Reply:
> Agree but the role of a controller is to validate inputs. Unless there is api controller somewhere is it's good enough for now. I'll add a TODO

Think this is not on the /** @var comment? not clear what you mean

Reply:
Sorry, I got it now